Makefile

```diff
@@ -64,16 +64,42 @@ pypi: test
 	python setup.py sdist
 	twine upload dist/*
 
+# todo: see (i)
 .PHONY: lint
 lint:
 	pip install tox
 	tox -e lint
 
+# todo: see (i)
 .PHONY: mypy
 mypy:
 	pip install tox
 	tox -e mypy
 
+# todo: see (i)
+.PHONY: qc-manifest
+qc-manifest:
+	tox -e manifest
+
+# todo: see (i)
+.PHONY: qc-flake8
+qc-flake8:
+	tox -e flake8
+
+# todo: see (i)
+.PHONY: qc-pytest
+qc-pytest:
+	tox -e py
+
+# todo: (i) How can I do this idempotently? Run checks without installing or deleting anything?
+# ...Right now I can't run this without .tox/py and .tox/mypy getting installed and removed.
+.PHONY: test-locally-all
+test-locally-all: qc-pytest mypy qc-flake8 lint qc-manifest
```

Comment:
@matentzn I wanted to be able to run all of the tests and other QC checks locally, rather than pushing to GitHub and waiting for the continuous integration checks to run, so I made these `make` recipes. What I'd like from your help with is:
1. A way to do that; to run them locally that works reliably, if you happen to know of one.
2. If you don't know of any solution to (1) offhand, I'd like the liberty to add some `make` recipes myself, as I've done here. I am having some difficulty getting them all to run using `.tox` reliably, the way the project is set up. So it may take some iteration on this. Refer to #257 to see some of the issues I'm having.

Reply:
Why can't you just run "tox"?